docs/home.md

```diff
@@ -0,0 +1,22 @@
+# Welcome to ![banner](assets/banner.png)
+
+!> TechPass is currently in active development. The information within this documentation will change without notice and may not up-to-date.
+
+## What is TechPass
+TechPass is an Identity & Access Management (IAM) and Single Sign On (SSO) Solution for services within the Singapore Government Technology Stack (SGTS).
+It is built to provide users with a seamless login experience across SG TechStack services and allow access control to be managed easily from a centralised location.
+
+Head over to our [Getting Started](quickstart) section to find out how you can start using TechPass.
+
+## Key Concepts
+TechPass is powered by [Microsoft Azure AD service](https://azure.microsoft.com/en-us/services/active-directory/). 
+It builds on top of Azure AD to cater to the specific requirements of the public service's development environment. 
+TechPass empowers engineering teams working on SGTS products to develop and rollout their services quickly by simplifying the amount of work needed to implement a new service.
+
+TechPass utilises popular open standards such as, [OAuth 2.0](https://oauth.net/2/) / [OIDC](https://openid.net/connect/) (OpenID Connect) 
+and [SAML 2.0](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html) (Security Assertion Markup Language), 
+for authentication and authorization.
+
+OAuth 2.0 setup service is available on TechPass while SAML service is available only by invitation. 
```

Comment:
* Can we mention TechPass portal specifically for OAuth 2.0 and provide the link?
* Instead of "invitation", maybe by "request" and also provide the link to the SR or steps on how to do it.

Reply:
done, I've created a section on ## TechPass Portal